ui/pages/snaps/snaps-list/index.scss

```diff
@@ -0,0 +1,52 @@
+.snaps {
```

Comment:
Same question here for the CSS

Reply:
As agreed on Slack, I asked design system about this. Once we have an answer, we can refactor in the next iteration. I imagine that we should have design system component that is a wrapper for this, but will see. 

I think this can be resolved for now?